paasta_tools/setup_kubernetes_job.py

```diff
@@ -103,43 +103,78 @@ def validate_job_name(service_instance: str) -> bool:
     return True
 
 
+def _bounce_service_instance(
+    deploy_strategy: str,
+    app: DeploymentWrapper,
+    kube_client: KubeClient,
+    existing_apps: dict,
+    existing_kube_deployments: set,
+) -> None:
+
+    if deploy_strategy == "brutal":
+        app.deep_delete(kube_client)
```

Comment:
I am wondering if we want to set grace_period_seconds as 0 for brutal bounce https://github.com/kubernetes-client/python/blob/master/kubernetes/docs/ExtensionsV1beta1Api.md#parameters-14